zips/zip-0246.rst

```diff
@@ -0,0 +1,566 @@
+
+A BLAKE2b-256 hash of the data from the burn fields of the transaction. For each tuple in
+the $\mathsf{assetBurn}$ set, the following elements are included in the hash::
+
+    T.4b.i : assetBase    (field encoding bytes)
+    T.4b.ii: valueBurn    (64-bit unsigned little-endian)
+
+The personalization field of this hash is set to::
+
+    "ZTxIdOrcBurnHash"
+
+In case the transaction does not perform the burning of any Assets (i.e. the
+$\mathsf{assetBurn}$ set is empty), the ``orchard_burn_digest`` is::
+
+    BLAKE2b-256("ZTxIdOrcBurnHash", [])
+
+The field encodings are specified in ZIP 230 [#zip-0230-orchard-asset-burn-field-encodings]_.
+
+
+T.5: issuance_digest
+````````````````````
+A BLAKE2b-256 hash of the following values ::
+
+   T.5a: issue_actions_digest    (32-byte hash output)
+   T.5b: issuanceValidatingKey   (field encoding bytes)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdSAIssueHash"
+
+In case the transaction has no issuance components, ``issuance_digest`` is::
+
+    BLAKE2b-256("ZTxIdSAIssueHash", [])
+
+The field encodings are specified in ZIP 230 [#zip-0230-transaction-field-encodings]_.
+
+T.5a: issue_actions_digest
+''''''''''''''''''''''''''
+A BLAKE2b-256 hash of Issue Action information for all Issuance Actions belonging to the transaction. For each Action, the following elements are included in the hash::
+
+   T.5a.i  : notes_digest            (32-byte hash output)
+   T.5a.ii : assetDescHash           (field encoding bytes)
+   T.5a.iii: flagsIssuance           (1 byte)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdIssuActHash"
+
+The field encodings are specified in ZIP 230 [#zip-0230-issue-actions-field-encodings]_.
+
+T.5a.i: issue_notes_digest
+..........................
+A BLAKE2b-256 hash of Note information for all Notes belonging to the Issuance Action. For each Note, the following elements are included in the hash::
+
+   T.5a.i.1: recipient                    (field encoding bytes)
+   T.5a.i.2: value                        (field encoding bytes)
+   T.5a.i.3: rho                          (field encoding bytes)
+   T.5a.i.4: rseed                        (field encoding bytes)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdIAcNoteHash"
+
+In case the transaction has no Issue Notes, ``issue_notes_digest`` is::
+
+    BLAKE2b-256("ZTxIdIAcNoteHash", [])
+
+The field encodings are specified in ZIP 230 [#zip-0230-issue-notes-field-encodings]_.
+
+T.6: memo_digest
+````````````````
+A BLAKE2b-256 hash of the following values ::
+
+   T.6a: nonce                 (field encoding bytes)
+   T.6b: memo_chunks_digest    (32-byte hash output)
+
+The personalization field of this hash is set to::
+
+  "ZTxIdMemo___Hash" (3 underscore characters)
+
+In case the transaction has no memo chunks, ``memo_digest`` is::
+
+    BLAKE2b-256("ZTxIdMemo___Hash", [])
+
+The field encodings are specified in ZIP 230 [#zip-0230-issuance-field-encodings]_.
```

Comment:
I think this should correspond to this section of ZIP 231 (which defines ``nonceOrHash`` rather than ``nonce`` though)
```suggestion
The field encodings are specified in ZIP 231 [#zip-0231-encoding-in-transactions]_.
```